### PR TITLE
footer text updates

### PR DESCRIPTION
**Pull request by schlosser**

### CODE REVIEW

File: src/common/metrics/contact_tracing.ts [hunk 1]
```diff
@@ -100,8 +100,8 @@ export function contactTracingStatusText(projection: Projection) {
   const outcomesAtLevel = levelText(
     level,
     'When this level of tracing is coupled with widely available testing, COVID can be contained without resorting to lockdowns.',
-    'When this level of tracing is coupled with widely available testing, COVID can be contained without resorting to lockdowns.',
-    'Tracing is largely ineffective relative to these levels of case growth and does little to contain the spread of COVID.',
+    'At these lower levels of tracing, it is unlikely Mississippi will be able to successfully identify and isolate sources of disease spread and prevent new outbreaks.',
```
Comment (goldblatt) [inline src/common/metrics/contact_tracing.ts:103]: {locationName}

File: src/components/LocationPage/ChartsHolder.tsx [hunk 2]
```diff
@@ -190,30 +190,29 @@ const ChartsHolder = (props: {
...
-                    an average of 10 contact tracers per new case, as well as
-                    fast testing.{' '}
+                    that 80% of contacts for each new case must be traced within
+                    48 hours in order to contain COVID. Experts estimate that
+                    tracing each new case within 48 hours requires an average of
+                    10 contact tracers per new case, as well as fast testing.
+                    Our contact tracing data is sourced from{' '}
                     <a
-                      href="https://blog.covidactnow.org/modeling-metrics-critical-to-reopen-safely/"
+                      href="https://testandtrace.com/"
```
Comment (goldblatt) [inline src/components/LocationPage/ChartsHolder.tsx:201]: these got moved to the metric/contact_tracing.ts file

File: src/common/metrics/contact_tracing.ts (hunk 1 above)
Comment (schlosser) [reply src/common/metrics/contact_tracing.ts]: Fixed!

File: src/components/LocationPage/ChartsHolder.tsx (hunk 2 above)
Comment (schlosser) [reply src/components/LocationPage/ChartsHolder.tsx:201]: i think this can't move into `metric/contact_tracing.ts` because of hyperlinks

**Pull request merged**